package.json

```diff
@@ -16,19 +16,20 @@
     "screencast:error": "svg-term --cast jyu19xGl88FQ3poMY8Hbmfw8y --out screencast-error.svg --window --at 12000 --no-cursor",
     "test": "cd packages/react-scripts && node bin/react-scripts.js test --env=jsdom",
     "format": "prettier --trailing-comma es5 --single-quote --write 'packages/*/*.js' 'packages/*/!(node_modules)/**/*.js'",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "webpack-defaults": "webpack-defaults"
   },
   "devDependencies": {
-    "eslint": "4.15.0",
+    "eslint": "4.19.0",
     "execa": "^0.9.0",
-    "husky": "^0.13.2",
-    "lerna": "2.6.0",
-    "lerna-changelog": "^0.6.0",
-    "lint-staged": "^3.3.1",
+    "husky": "^0.14.3",
+    "lerna": "2.9.0",
+    "lerna-changelog": "^0.7.0",
+    "lint-staged": "^7.0.0",
```

Comment:
I a bit concerned about bumping unrelated packages in this PR, especially when we're bumping major versions. Can you limit the package updates to the bare minimum so we can better evaluate this change?

Reply:
It's fine to leave these as-is with the exception of the `source-map` dependency I mentioned in my other comment.

Reply:
yeah i agree, it should have been handled seperately, and it still can.

I can make a bump PR and then rebase this branch on top of it if you want to.